add timezone and locale description

YuliyaPoznyak opened the pull request.

georged reviewed (changes requested):
@YuliyaPoznyak you added timezone and locale in the wrong place. Should be the list of properties. You can add an example of using user.timezone to convert a random date eg "now" to user's timezone. 

https://docs.alexacrm.com/twig/introduction/#access-the-current-user-record

georged reviewed (changes requested):
@YuliyaPoznyak 

> - `timezone`-- Overrides the date using the current user's timezone.
> - `locale` -- Overrides the date using the current user's locale.

This is wrong. These properties do not override anything, they simply return timezone or locale for the current user. You need to explain what format do they return the information in (e.g. IANA timezone, etc, maybe even point to the public source like https://en.wikipedia.org/wiki/List_of_tz_database_time_zones and its locale equivalent) and provide a sample of a return value, e.g. "Australia/Sydney"

You also need to assert if they can be null.

YuliyaPoznyak commented:
@georged I changed the description for the timezones, created a new page to display individual field types using twig (to make the pages more related in meaning), added the description from the tasks from the board there (based on how it currently works). I can move some of the material to KB or break the new page into separate articles.
